### PR TITLE
Fix issues with updating template images

### LINKED ISSUE #323: Soft button content is misaligned in an alert
## Bug Report
If a soft button in an alert has both images and text, the text extends outside the bounds of the button. This was seen on Firefox web browser (Mozilla Firefox 83.0)

### Reproduction Steps
1. Create an alert with two soft buttons. One button is text-only ("Ok") and the second button has text and an image ("Gotcha" with the speech bubble icon). 
1. Send the alert request. 

#### Expected Result
The soft button text and image is center aligned inside the button.

#### Actual Result
The soft button text and image are misaligned. Text extends outside the bounds of the soft button. 

Screen Shot 2020-11-18 at 2 05 46 PM

### PR DESCRIPTION
**Pull request by jacobkeeler**
Fixes #323 

Summary of Changes:
- Fixes warnings in `Image.js`
- Fixes issue where template images were cached and never updated
- Fixes issues with scaling when rerendering template images

### CODE REVIEW

**Review by iCollin — COMMENTED** [review]
Testing went well, I couldn't find any BOTH softbuttons overflowing.

This branch needs to be rebased on develop since we squashed the commits from the PR templates.

**Comment by jacobkeeler** [comment]
@iCollin Rebased commits, unfortunately it did require a force-push, but the changes were directly cherry-picked from the 3 original commits

**Review by iCollin — APPROVED** [review]
Testing went well, approved.

One question I have though, do you think we should leave the default canvas size as 300? I mistyped my image name at first, and the text was forced outside of an alert softbutton because by default the canvas was 300x75. This would also apply in the case an image upload failed. Would it make sense to use a smaller default canvas size?

File: src/js/Templates/Shared/Image.js [hunk 1]
```diff
@@ -7,13 +7,21 @@ class Image extends React.Component {
     constructor(props) {
         super(props);
         this.state = {error: false, refreshed: false};
-        this.canvasStyle = {
+        this.customCanvasStyle = {
```
[the remark this answers is not in the document]
Comment (JackLivio) [reply src/js/Templates/Shared/Image.js:10]: @iCollin Referring to canvasStyle before the changes in the pr, that object was created to store the canvasContainer height and width. There was a strange issue in safari where the canvasContainer height would increase when the image in a button that was clicked on.

**Review by iCollin — APPROVED** [review]
I could not reproduce https://github.com/smartdevicelink/generic_hmi/issues/293 on this branch. I think we can remove `customCanvasStyle`.

**Review by iCollin — APPROVED** [review]

**Pull request merged**